instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/builder/internal/DefaultHttpClientInstrumenterBuilder.java

```diff
@@ -218,8 +219,14 @@ public Instrumenter<REQUEST, RESPONSE> build() {
     return builder.buildInstrumenter(SpanKindExtractor.alwaysClient());
   }
 
-  public OpenTelemetry getOpenTelemetry() {
-    return openTelemetry;
+  public ContextPropagators getPropagators() {
+    return openTelemetry.getPropagators();
+  }
+
+  public <BUILDERREQUEST, BUILDERRESPONSE>
+      InstrumenterBuilder<BUILDERREQUEST, BUILDERRESPONSE> instrumenterBuilder(
+          SpanNameExtractor<? super BUILDERREQUEST> spanNameExtractor) {
+    return Instrumenter.builder(openTelemetry, instrumentationName, spanNameExtractor);
```

Comment:
this method is confusing, I finally understood it, but wonder if there's anything we can do to make it clearer

Reply:
maybe `toInstrumenterBuilder`?